source-mongodb/pull.go

```diff
@@ -797,3 +809,19 @@ func idToString(value interface{}) string {
 	}
 	return string(j)
 }
+
+type buildInfo struct {
+	Version       string `bson:"version"`
+	MaxObjectSize int    `bson:"maxBsonObjectSize"`
```

Comment:
This is always 16MB, no? From their docs:

> BSON Document Size
> The maximum BSON document size is 16 megabytes.
https://www.mongodb.com/docs/manual/reference/limits/#bson-documents

Reply:
On extremely old versions it was 4MB, but those are probably so old they would have other problems running. We'd get the size limit on any "document too large" errors, so I'll remove this from the logging.